js/foundation.tabs.js

```diff
@@ -98,8 +104,14 @@ class Tabs extends Plugin {
      //current context-bound function to open tabs on page load or history hashchange
     this._checkDeepLink = () => {
       var anchor = window.location.hash;
+
+      // if there is no anchor, return to the initial tab
+      if (!anchor.length && this._initialAnchor) {
+        anchor = this._initialAnchor;
+      }
+
       //need a hash and a relevant anchor in this tabset
-      if(anchor.length) {
+      if (anchor.length) {
         var anchorNoHash = (anchor.indexOf('#') >= 0 ? anchor.slice(1) : anchor);
         var $link = this.$element.find(`[href$="${anchor}"],[data-tabs-target="${anchorNoHash}"]`).first();
         if ($link.length) {
```

Comment:
In the course of testing the tabs I noticed the focus style remains when using history back.

![tabs_deep_linking_480px_low](https://user-images.githubusercontent.com/19912475/44781330-5e3eb200-ab84-11e8-8390-34dd1439f7c0.gif)

What do you think about auto focusing the link to fix this?

```javascript
if ($link.length) {
  $link.focus();
  this.selectTab($(anchor), true);
```

Reply:
This is an other issue, not related to this pull request. Why not, but I wonder what's the impact on accessibility to move the focus like this.